### DISCUSSION TITLE
deprecated icon codes used through-out the code

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [character-map-uwp/Character-Map-UWP#274] [Ideas]
Microsoft had deemed a range of icon codes (E0** - E5**) as deprecated and should not be used any more. There are replacement code points for them in a higher range. For example: **E115** ("Setting", cog-wheel) should be replaced by **E713**.

![message](https://user-images.githubusercontent.com/65828559/111816539-3f889e00-88dd-11eb-85f6-e9b0f0f65256.png)
![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/65828559/a85a69c6-2e85-471b-8203-78757d5c28e8)

Reason is explained in this https://github.com/microsoft/microsoft-ui-xaml/pull/8379#issuecomment-1593855795

**Comment by Jay-o-Way** [comment]
@JohnnyWestlake and why not?

**Comment by JohnnyWestlake** [comment]
I have no qualms if someone else wants to go ahead and address this, however as the deprecation was done as a safety net for scenarios that will not be encountered in this app and my time to work on this is fairly limited, I probably won't tackle it myself.
